### PR TITLE
Loader Changes Commit -DO NOT MERGE

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**
Loader changes

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-base/.content.xml [hunk 1]
```diff
@@ -3,5 +3,5 @@
...
+    embed="[core.wcm.components.accordion.v1,core.wcm.components.tabs.v1,core.wcm.components.carousel.v1,core.wcm.components.image.v2,core.wcm.components.breadcrumb.v2,core.wcm.components.search.v1,core.wcm.components.form.text.v2,hclecomm.grid,hclecomm.title,hclecomm.text,hclecomm.button,hclecomm.accordion,hclecomm.image,hclecomm.teaser,hclecomm.carousel,hclecomm.tabs,hclecomm.footerContainer,hclecomm.navigation,hclecomm.search,hclecomm.minicart,hclecomm.productdetails,hclecomm.cartProductDisplay,hclecomm.productcart,hclecomm.productfilter,hclecomm.checkoutform,hclecomm.signin,hclecomm.signup,hclecomm.checkoutcarttotal,hclecomm.confirmation,hclecomm.site]"/>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-base/.content.xml:6]: @DeepaliAgrwl  : hclecomm.site client lib is part of page policy . why has this been included in hclecomm.base?

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-site/js/loader.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    if(flag===true){
+        body.style.overflow="hidden";
+        body.innerHTML+=`<div class="overlay" style="height:${window.innerHeight}px"><div class="loader"></div></div>`
+    }else{
+        body.style.removeProperty("overflow");
+        let overlay= document.querySelector(".overlay");
+        body.removeChild(overlay);
+    }
+}
+/*setTimeout(function(){
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-site/js/loader.js:13]: @DeepaliAgrwl @ashishgoyalweb : pl remove js comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-site/js/loader.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+function loader(flag){
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-site/js/loader.js:1]: @DeepaliAgrwl @ashishgoyalweb : where is the functional call happening ?

**Review by rishabh-behl — COMMENTED** [review]
@DeepaliAgrwl @ashishgoyalweb : Pl check comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-base/.content.xml (hunk 1 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-base/.content.xml:6]: @rishabh-behl  In page policy we have included only clientlib.base and clientlib.dependencies library. That is why I have embedded it with clientlib.base

**Comment by DeepaliAgrwl** [comment]
Closing this PR s the changes has been committed using the new branch for site Loader

**Pull request closed without merging**